File: src/main/java/com/sport/service/web/controllers/AlertController.java
//package com.sport.service.web.controllers;
//
//import com.sport.service.bot.TelegramMessageSender;
//import org.springframework.beans.factory.annotation.Value;
//import org.springframework.web.bind.annotation.*;
//
//@RestController
//@RequestMapping("/alerts")
//public class AlertController {
//
//    private final String mainAdminId;
//    private final TelegramMessageSender sender;
//
//    public AlertController(
//            @Value("${telegram.mainAdminId}") String mainAdminId,
//            TelegramMessageSender sender
//    ) {
//        this.mainAdminId = mainAdminId;
//        this.sender = sender;
//    }
//
//    @PostMapping("/alerts")
//    public void receiveAlert(@RequestBody String rawJson) {
//        sender.sendMessageWithoutPhoto(Long.parseLong(mainAdminId), rawJson);
//    }
//}

File: src/main/kotlin/com/sport/service/web/controllers/AlertController.kt
//package com.sport.service.web.controllers
//
//import com.sport.service.bot.TelegramMessageSender
//import org.springframework.beans.factory.annotation.Value
//import org.springframework.web.bind.annotation.PostMapping
//import org.springframework.web.bind.annotation.RequestBody
//import org.springframework.web.bind.annotation.RequestMapping
//import org.springframework.web.bind.annotation.RestController
//
//@RestController
//@RequestMapping("/alerts")
//class AlertController(
//    @Value("\${telegram.mainAdminId}") private val mainAdminId: String,
//    private val sender: TelegramMessageSender
//) {
//
//    @PostMapping("/alerts")
//    fun receiveAlert(@RequestBody rawJson: String) {
//        sender.sendMessageWithoutPhoto(mainAdminId.toLong(), rawJson)
//    }
//}